Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -2563,14 +2566,15 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
         WASM_PARSER_FAIL_IF(argumentCount > m_expressionStack.size(), "call_ref expects ", argumentCount, " arguments, but the expression stack currently holds ", m_expressionStack.size(), " values");
 
         Vector<ExpressionType> args;
-        WASM_PARSER_FAIL_IF(!args.tryReserveCapacity(argumentCount + 1), "can't allocate enough memory for ", argumentCount, " call_indirect arguments");
```

Comment:
This looks like it is not needed, right?

Reply:
We need to check this first to allocate capacity with fail-safe way. And then we should call resize to make Vector size matched (not capacity).

Reply:
I meant the +1

Reply:
Yeah, looks like we do not need it.